certified-connectors/IN-D Insurance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
@gowthamd3 This swagger is not up to date as per the artifacts you have provided in ISV Studio. Please remove the swagger and apiProperties.json files from the PR and add the connector artifacts you have provided in the ISV Studio.

Reply:
Hi Srikanth, Updated the swagger file and apiProperties.json file.